data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -522,6 +526,10 @@ impl TraceExporter {
                     }
                     match response.into_body().collect().await {
                         Ok(body) => {
+                            info!(
+                                trace_count = trace_count,
```

Comment:
I know nothing about Rust, so forgive my ignorance, but is it on purpose you assign this variable to itself? I see you do it elsewhere so I assume it's on purpose :)

Reply:
that's a name-value pair, could be shorthand

https://docs.rs/tracing/latest/tracing/#recording-fields